vignettes/articles/05-data-cleaning.Rmd

```diff
@@ -210,31 +209,29 @@ If you want to summarize at a finer/coarser scale, you can also create other gro
 The simplest alternative is a rounded timestamp variable; for example by using `mutate(ts.h = plyr::round_any(ts, 3600)`.
 Other options are to just use date (e.g `date = as_date(ts)`).
 
-Here we create a summary data frame that we can filter to produce different plots
+Here we create a summary data frame that we can filter to produce different plots:
 
 ```{r eval = TRUE}
-df.plot <- tbl.alltags.sub %>%
+df.summary <- tbl.alltags.sub %>%
   filter(tagProjID == 176, # keep only tags registered to the sample project
          !is.na(recvDeployLat) | !(recvDeployLat == 0)) %>% # drops data without lon/lat
   group_by(motusTagID, runID, recvDeployName, ambigID, 
            tagDepLon, tagDepLat, recvDeployLat, recvDeployLon) %>%
   #summarizing by runID to get max run length and mean time stamp:
-  summarize(max.runLen = max(runLen, na.rm = TRUE), 
-            ts = mean(ts, na.rm = TRUE), .groups = "drop") %>% 
+  summarize(max.runLen = max(runLen), 
+            ts = mean(ts)) %>% 
   arrange(motusTagID, ts) %>%
-  collect() %>%
-  mutate(ts = plyr::round_any(ts, 3600),
-         ts = as_datetime(ts, tz = "UTC", origin = "1970-01-01"))
+  collect()
 ```
 
 We would initially plot a subset of tags by either latitude or longitude[^1], to get an overview of where there might be issues. 
 Here, to simplify the example, we plot only six tags. 
 We avoid examining the ambiguous tags for now. 
 
 ```{r}
-ggplot(data = filter(df.plot,
+ggplot(data = filter(df.summary,
                      motusTagID %in% c(16011, 16035, 16036, 16037, 16038, 16039)), 
-       aes(x = ts, y = recvDeployLat)) +
+       aes(x = as_datetime(ts), y = recvDeployLat)) +
```

Comment:
We shouldn't need `as_datetime` here anymore if we reinstate in the summarizing code above.